tools/generator/CodeGeneratorContext.cs

```diff
@@ -16,5 +16,17 @@ public class CodeGeneratorContext
 		string ContextMethodString => ContextMethod != null ? "in method " + ContextMethod.Name + " " : null;
 		string ContextTypeString => ContextType != null ? "in managed type " + ContextType.FullName : null;
 		public string ContextString => ContextFieldString + ContextMethodString + ContextTypeString;
+
+		public string GetContextTypeMember ()
+		{
+			var output = ContextType?.FullName ?? string.Empty;
+
+			if (ContextMethod != null) {
+				output += $"{ContextMethod.Name} ({string.Join (", ", ContextMethod?.Parameters.Select (p => p.RawType).ToArray ())})";
```

Comment:
The `p.RawType` here confuses me.

For starers, I can't find it?

```bash
% git grep 'RawType' 
tests/Xamarin.Android.Tools.ApiXmlAdjuster-Tests/api-24.xml.in:        name="getRawType"
```

(Yet the PR *builds*, which makes this even *more* confusing to me.)

Secondly, do we want the "signature" to contain *Java* names or *C#* names?  I'd think we'd want the `Parameter.Type` property? 

https://github.com/xamarin/java.interop/blob/f6c12ba30048748572af57654b218c60bc9ff95f/tools/generator/Java.Interop.Tools.Generator.ObjectModel/Parameter.cs#L222-L226

Reply:
This point of the process is looking up Java types (`type`) and converting them to C# types (`sym`).  So `sym` is still `null`, causing an `NRE` if we use `Parameter.Type`.